client/my-sites/customer-home/cards/tasks/deprecate-editor/index.jsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { preventWidows } from 'lib/formatting';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import Task from '../task';
+
+import { setSelectedEditor } from 'state/selected-editor/actions';
+import {
+	composeAnalytics,
+	recordGoogleEvent,
+	recordTracksEvent,
+	withAnalytics,
+	bumpStat,
+} from 'state/analytics/actions';
+import getGutenbergEditorUrl from 'state/selectors/get-gutenberg-editor-url';
+import blockEditorImage from 'assets/images/illustrations/block-editor-fade.svg';
+
+const DeprecateEditor = ( { siteId, gutenbergUrl, optIn } ) => {
+	const translate = useTranslate();
+	const actionCallback = () => {
+		optIn( siteId, gutenbergUrl );
+	};
+	return (
+		<Task
+			title={ translate( 'The Block Editor is coming' ) }
+			description={ preventWidows(
+				translate(
+					'Try the Block Editor now before it is enabled for everyone on {{strong}}May 22{{/strong}}. {{a}}Read more{{/a}}',
```

Comment:
The date is supposed to be bold according to designs but font-weight has no effect on the font used here:

<img width="581" alt="Screen Shot 2020-04-30 at 4 10 46 PM" src="https://user-images.githubusercontent.com/3629020/80671195-41971200-8afd-11ea-8530-ef2aa170ae88.png">

any ideas on how to get a bolder font here?

Reply:
Probably related to the existing issue in Chrome 81 that break the ability to bold system fonts: https://www.coywolf.news/webmaster/chrome-81-breaks-system-fonts-bold/